CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-09-25
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Release notes 
+* Behavioral changes
+  * If your VM SKU does not support ephemeral or PremiumSSD OS disks, AKS will now use StandardSSD as the default OS disk type. Before, only VM SKUs that did not support ephemeral OS disks had this default.
+
+* Features
+  * Load Balancer TCP reset can now be desabled with a service annotaiton *"service.beta.kubernetes.io/azure-load-balancer-disable-tcp-reset=true"*. [Upstream link](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4520) in preview
```

Comment:
Image Integrity is now Public Preview: https://learn.microsoft.com/azure/aks/image-integrity

Reply:
added with description